README.md

```diff
@@ -1 +1,14 @@
-# WIP
+
+# build docker image
+
+```
+docker build -t asyncapi-editor .
+```
+
+# run docker image
+
+```
+docker run -d --name asyncapi_editor -p 83:3000 asyncapi-editor:latest
```

Comment:
This line is not true now. Port should be 5000.

Reply:
fixed